nova/virt/libvirt/driver.py

```diff
@@ -6009,12 +6016,30 @@ def _get_guest_config(self, instance, network_info, image_meta,
         for config in storage_configs:
             guest.add_device(config)
 
+        net_index = 0
         for vif in network_info:
             config = self.vif_driver.get_config(
                 instance, vif, image_meta,
                 flavor, virt_type, self._host)
             guest.add_device(config)
 
+            if vif.get("vnic_type") == "virtio-forwarder":
+                # Enable page-per-vq for virtio-forwarder interface
+                args_dict = {"page-per-vq": "on"}
+                if (vif.get("network") and vif.get("network").get("meta") and
+                        vif.get("network").get("meta").get("mtu")):
+                    # Set mtu_host in xml file for virtio-forwarder interface
+                    args_dict["host_mtu"] = vif["network"]["meta"]["mtu"]
```

Comment:
I think the host_mtu is a WA. Please add a note that we need to revisit if it need or not?